config/settings.yaml

```diff
@@ -0,0 +1 @@
+run_timeout: 3600
```

Comment:
this should not be included in the PR, this file should be in .gitignore

Reply:
Do you mean the "settings.yaml" file?